templates/focus-ios.xliff

```diff
@@ -294,6 +306,22 @@
         <source>Search for %@</source>
         <note>Label displayed for search button when typing in the URL bar</note>
       </trans-unit>
+      <trans-unit id="URL.trackersBlockedLabel">
+        <source>Trackers blocked</source>
+        <note>Text for the URL bar showing the number of trackers blocked on a webpage.</note>
+      </trans-unit>
+      <trans-unit id="contextMenu.copyImageTitle">
+        <source>Copy Image</source>
+        <note>Text for the context menu when a user wants to copy an image after long pressing it.</note>
+      </trans-unit>
+      <trans-unit id="contextMenu.saveImageTitle">
+        <source>Save Image</source>
+        <note>Text for the context menu when a user wants to save an image after long pressing it.</note>
+      </trans-unit>
+      <trans-unit id="externalAppLinkWithAppName.messageTitle">
+        <source>%1$@ wants to open %2$@</source>
+        <note>Dialog title used for opening an external app from Focus</note>
```

Comment:
I think @bkmunar did say he mistakenly didn't to the last PR from master - sorry!